test/e2e-common.sh

```diff
@@ -89,6 +90,9 @@ function knative_eventing() {
     echo ">> Install Knative Eventing from latest - ${EVENTING_CONFIG}"
     kubectl apply -f "${EVENTING_CONFIG}/eventing-crds.yaml"
     kubectl apply -f "${EVENTING_CONFIG}/eventing-core.yaml"
+	# we need cert-manager installed to be able to create the issuers
+	kubectl apply -Rf "${CERTMANAGER_CONFIG}"
```

Comment:
Don't we need to apply cert-manager regardless of being a release or non release branch?

Reply:
Good point!